flocker/acceptance/testtools.py

```diff
@@ -771,15 +781,20 @@ def get_test_cluster(node_count=0):
                        "{existing} node(s) are set.".format(
                            necessary=node_count, existing=len(agent_nodes)))
 
+    certificates_path = FilePath(
+        environ["FLOCKER_ACCEPTANCE_API_CERTIFICATES_PATH"])
     cluster = Cluster(
         control_node=ControlService(address=control_node),
-        nodes=[]
+        nodes=[],
+        treq=treq_with_authentication(reactor, certificates_path)
     )
 
     # Wait until nodes are up and running:
     def nodes_available():
         d = cluster.current_nodes()
-        d.addCallback(lambda (cluster, nodes): len(nodes) >= node_count)
+        d.addCallbacks(lambda (cluster, nodes): len(nodes) >= node_count,
+                       # Control service may not be up yet, keep trying:
+                       lambda failure: False)
```

Comment:
Worth logging this failure, probably, particularly since `loop_until` has no timeout.